daal4py/sklearn/svm/_svm_0_23.py

```diff
@@ -389,6 +389,13 @@ def fit(self, X, y, sample_weight=None):
             raise TypeError("Sparse precomputed kernels are not supported.")
         self._sparse = sparse and not callable(self.kernel)
 
+        if hasattr(self, 'decision_function_shape'):
+            if self.decision_function_shape not in ('ovr', 'ovo'):
+                raise ValueError(
+                    f"decision_function_shape must be either 'ovr' or 'ovo', "
+                    f"got {self.decision_function_shape}."
```

Comment:
Maybe add quotes like in previous line: ``f"got '{self.decision_function_shape}'."``

Reply:
copied verbatim from upstream.